build_datasets.R

```diff
@@ -0,0 +1,187 @@
+
+# Load libraries ----------------------------------------------------------
+
+
+library(dplyr)
+library(dbplyr)
+library(sconn) # https://github.com/The-Strategy-Unit/sconn
+
```

Comment:
```suggestion

```

If we're namespacing consistently - let's drop these. 

Can always link to {sconn} isntall in README